src/utils/signals.js

```diff
@@ -52,7 +52,8 @@ export class Signals {
    * @return {number|!Error|null}
    */
   get(name) {
-    return this.map_[name] || null;
+    const v = this.map_[name];
+    return v === 0 ? v : v || null;
```

Comment:
```suggestion
    return v == null ? null : v;
```

To match with the [nullish coalescing operator](https://babeljs.io/repl/#?babili=false&browsers=Safari%20>%209&build=&builtIns=false&spec=false&loose=false&code_lz=MYewdgzgLgBAbjAvDADAbgFAIPzZmAVwBsi0g&debug=false&forceAllTransforms=false&shippedProposals=false&circleciRepo=&evaluate=false&fileSize=false&timeTravel=false&sourceType=module&lineWrap=true&presets=es2015-loose&prettier=true&targets=&version=7.5.5&externalPlugins=%40babel%2Fplugin-proposal-nullish-coalescing-operator%407.4.4): `v ?? null`

Reply:
Done!